com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v3_1/ServiceKeysCloudModelBuilder.java

```diff
@@ -0,0 +1,39 @@
+package com.sap.cloud.lm.sl.cf.core.cf.v3_1;
+
+import static com.sap.cloud.lm.sl.cf.core.util.CloudModelBuilderUtil.isActive;
+import static com.sap.cloud.lm.sl.cf.core.util.CloudModelBuilderUtil.isService;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.cloudfoundry.client.lib.domain.ServiceKey;
+
+import com.sap.cloud.lm.sl.cf.core.helpers.v1_0.PropertiesAccessor;
+import com.sap.cloud.lm.sl.common.ContentException;
+import com.sap.cloud.lm.sl.mta.model.v1_0.DeploymentDescriptor;
+import com.sap.cloud.lm.sl.mta.model.v1_0.Resource;
+
+public class ServiceKeysCloudModelBuilder extends com.sap.cloud.lm.sl.cf.core.cf.v1_0.ServiceKeysCloudModelBuilder {
+    
+    private DeploymentDescriptor deploymentDescriptor;
```

Comment:
maybe set them to protected in the parent class instead of re-declaring them here?